velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,28 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp = [&](const std::string& functionName,
+                                            int64_t expectedTimestamp,
+                                            const std::string& timeZone) {
+    VectorPtr expected =
+        makeTimestampWithTimeZoneVector(expectedTimestamp, timeZone.c_str());
+    VectorPtr actual =
+        evaluateWithTimestampWithTimezone(functionName, 0, "+00:00");
+
+    // Accept 5 milliseconds lag since sometimes the machine is slow.
+    ASSERT_TRUE(actual->compare(expected.get(), 0, 0) < 5)
+        << ": expected " << expected->toString(0) << ", but got "
+        << actual->toString(0)
+        << ", which is larger than the 5 milliseconds acceptable lag.";
+  };
+
+  evaluateCurrentTimestamp(
```

Comment:
Last comment : Did you think about testing other timezones also ?

The rest of the code looks good otherwise.

Reply:
The timezone can be set to the session timezone. If there's no session timezone, fallback to 0 (GMT). For the test I wrote, I mainly focused on the 'timestamp' part since it is the main part returned by the 'current_timestamp' function, so the timezone is all set to "+00:00". 
I was looking for a way to set the session timezone in the test case, but I didn't find it.

Reply:
Ok, never mind. I rewrote the test case and added the setting of the session timezone and its verification. So it can check the time zone now. Please review it again when you have time. Thanks. @aditi-pandit